repos/system_upgrade/el8toel9/actors/checkarmbootloader/actor.py

```diff
@@ -0,0 +1,22 @@
+import leapp.libraries.actor.checkarmbootloader as checkarmbootloader
+from leapp.actors import Actor
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckArmBootloader(Actor):
+    """
+    Inhibit ARM system upgrades on path with incompatible kernel/bootloader
+
+    Due to an incompatibility of RHEL8 bootloader with newer versions of kernel
+    on RHEL9 since version 9.5, the upgrade cannot be performed as the old
+    bootloader cannot load the new kernel when entering the interim phase.
```

Comment:
It's better to make it clear that we do not expect this to be there forever

```suggestion
    bootloader cannot load the new kernel when entering the interim phase.
    
    This is temporary workaround until the issue is resolved.
```

Reply:
Sure! Fixed